lib/DDG/Spice/ClojurePackagesFromClojars.pm

```diff
@@ -0,0 +1,43 @@
+package DDG::Spice::ClojurePackagesFromClojars;
```

Comment:
Lets simplify this name! `Clojars` (like Npm, RubyGems, Hoogle) is fine, or `ClojurePackages` (similar to SublimePackages)